cvpysdk/commcell.py

```diff
@@ -67,9 +67,11 @@ class instance
 
     download_software()         --  triggers the Download Software job with the given options
 
-    sync_remote_cache()         --  syncs remote cache

```

Comment:
Looks like we have a lot of new changes on top of the file present on the master branch. You might have to update the local repository with dev branch before uploading the changes. 

Please let us know if you prefer us to add this enhancement.

Thanks.

Reply:
Yes, I can retest a client install with starting with dev branch, then add my changes to dev branch and redo PR.  Thanks.

Reply:
Thanks.